client/js/src/test/scala/org/http4s/client/scaffold/ServerScaffold.scala

```diff
@@ -16,6 +16,23 @@
 
 package org.http4s
 package client
+package scaffold
+
+/*
+ * Copyright 2014 http4s.org
```

Comment:
Hmm, we already have a header in this file. Probably this came after moving file?

Reply:
Thanks. Corrected.